assets/components/atoms/picture/picture-cover.twig

```diff
@@ -1,18 +1,27 @@
 <picture>
-  <source
-    media="(min-width: 1140px)"
-    srcset="https://via.placeholder.com/1920x1080.jpg 1x,https://via.placeholder.com/2880x1620.jpg 2x">
-  <source
-    media="(min-width: 960px)"
-    srcset="https://via.placeholder.com/1140x641.jpg 1x,https://via.placeholder.com/2280x1283.jpg 2x">
-  <source
-    media="(min-width: 720px)"
-    srcset="https://via.placeholder.com/960x540.jpg 1x,https://via.placeholder.com/1920x1080.jpg 2x">
-  <source
-    media="(min-width: 541px)"
-    srcset="https://via.placeholder.com/720x405.jpg 1x,https://via.placeholder.com/1440x810.jpg 2x">
-  <source
-    media="(max-width: 540px)"
-    srcset="https://via.placeholder.com/540x304.jpg 1x,https://via.placeholder.com/1080x608.jpg 2x">
-  <img src="https://via.placeholder.com/1920x1080.jpg" class="img-fluid" alt="Cover description">
+    {% if img is not defined %}
+    
+      <source
+        media="(min-width: 1140px)"
+        srcset="https://via.placeholder.com/1920x1080.jpg 1x,https://via.placeholder.com/2880x1620.jpg 2x">
+      <source
+        media="(min-width: 960px)"
+        srcset="https://via.placeholder.com/1140x641.jpg 1x,https://via.placeholder.com/2280x1283.jpg 2x">
+      <source
+        media="(min-width: 720px)"
+        srcset="https://via.placeholder.com/960x540.jpg 1x,https://via.placeholder.com/1920x1080.jpg 2x">
+      <source
+        media="(min-width: 541px)"
+        srcset="https://via.placeholder.com/720x405.jpg 1x,https://via.placeholder.com/1440x810.jpg 2x">
+      <source
+        media="(max-width: 540px)"
+        srcset="https://via.placeholder.com/540x304.jpg 1x,https://via.placeholder.com/1080x608.jpg 2x">
+      <img src="https://via.placeholder.com/1920x1080.jpg" class="img-fluid" alt="Cover description">
+
+      {% else %}
+
+      <img src="{{img}}" class="img-fluid" alt="Cover description">
```

Comment:
That's a compromise to avoid being forced to generate multiple sizes for example images. I thought that for examples purposes, this would suffice. The only downside is that images will be too big on responsive mode.

We could also just always leave the sources, so that on desktop we have a nice example, and on mobiles it fallbacks to the example images, to make sure the integrators think about these